app/services/oligrapher_assets_service.rb

```diff
@@ -55,6 +55,7 @@ def run
         build_cmd << '--env.api_url=http://127.0.0.1:8081'
       else
         build_cmd << '--env.production'
+        build_cmd << '--env.public_path=/oligrapher/'
```

Comment:
Why does this path need to be changed? /oligrapher doesn't exist on our production server and right now the files are built correct...

Reply:
The public path to the oligrapher build is currently `https://littlesis.org/oligrapher/oligrapher-{version}.js`. Without providing an explicitly `publicPath` option to webpack it will by default look for chunks at the same base path as the page itself, which in most cases is `/oligrapher/`, which is correct, but for secret share links it will look for the chunks in `/oligrapher/{id}/share/`, which will break the page. So we provide an explicit `publicPath` to avoid this error.

Reply:
(And if we ever create additional routes for accessing oligrapher that add subpaths to the base `/oligrapher/`, they'll also need this configuration.)

Reply:
thanks, I had thought public path was the location on disk.